omod/src/main/java/org/openmrs/module/nigeriaemr/ndrfactory/PMTCTDictionary.java

```diff
@@ -744,6 +772,182 @@ public InfantPCRTestingType createInfantPcr(Patient pts, Encounter enc, List<Int
         return maternalCohortTypes.isEmpty() ? null :  maternalCohortTypes;
     }
 
+    public PMTCTHTSType createPMTCTHTS(List<Encounter> pmtctHTSEncounters){
+        Encounter pmtctHTSEncounter = pmtctHTSEncounters
+                .stream()
+                .filter(c -> c.getForm().getId() == 54)
+                .collect(Collectors.toList()).stream().findFirst().get();
+        PMTCTHTSType pmtcttHTSType = new PMTCTHTSType();
+        PMTCTClinicalTBScreeningType pmtctClinicalTBScreeningType = new PMTCTClinicalTBScreeningType();
+        TestResultType testResultType = new TestResultType();
+        if (pmtctHTSEncounter != null ) {
```

Comment:
you're checking for null after you called it in line 776, if it can be null, line 776 will throw a null pointer exception, you should move the check up